include/mysql_com.h

```diff
@@ -137,13 +137,9 @@
 #define FIELD_IS_DROPPED (1<< 26)       /* Intern: Field is being dropped */
 #define EXPLICIT_NULL_FLAG (1<< 27)     /* Field is explicitly specified as
                                            NULL by the user */
-<<<<<<< HEAD
 #define CLUSTERING_FLAG (1 << 28)       /* Field has a secondary clustering
                                         key */
-||||||| 0ed6d65f4c6
-=======
-#define SERIAL_FLAG (1<< 28)            /* Field is "SERIAL" */
->>>>>>> 0cd98bdf981583a1cf4cb526581fc16e23bb839b^
+#define SERIAL_FLAG (1<< 29)            /* Field is "SERIAL" */
```

Comment:
Based on `testclient/mysql_client_test.c`, it looks like these flags are sent to the client.

I see that right now the client doesn't do anything with this flag, but I'm still wondering if this is a good choice here.

Anytime PS sets the CLUSTERING_FLAG, the oracle client will see a field set with the SERIAL_FLAG. (or a custom client based on libmysql)

Unless I missed something in the code, we do not write these flags to the disk, so we could easily change the id of CLUSTERING_FLAG instead?

Reply:
Yeah sure. If this is the case ,we can keep #define SERIAL_FLAG (1<< 28)  as is and increment ID of CLUSTERING_FLAG
as  #define CLUSTERING_FLAG (1 << 29)
@oleksandr-kachan  are you ok with this?

Reply:
Done.